app/Imports/XlsformTemplateLanguageImport.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\LanguageString;
+use App\Models\SurveyRow;
+use App\Models\LanguageStringType;
+use Maatwebsite\Excel\Concerns\ToModel;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+
+class XlsformTemplateLanguageImport implements ToModel, WithHeadingRow
+{
+    protected $templateId;
+    protected $templateLanguageId;
+
+    public function __construct($templateId, $templateLanguageId)
+    {
+        $this->templateId = $templateId;
+        $this->templateLanguageId = $templateLanguageId;
+    }
+
+    public function model(array $row)
```

Comment:
Normally, the model() method asks you to return a `new Model()` class. As you're conditionally creating the new model here based on whether the surrvey row and language string type exists, I would use `onRow()` instead: https://docs.laravel-excel.com/3.1/imports/model.html#handling-persistence-on-your-own